dt-core/admin/menu/tabs/tab-workflows.php

```diff
@@ -743,7 +760,7 @@ private function workflows_design_section_step3() {
                             </tr>
                             <tr id="workflows_design_section_step3_action_value_tr">
                                 <td>
-                                    value:
+                                    value: <a id="workflows_design_section_step3_help_text" type="hidden"></a>
```

Comment:
It's not a good practice to have an empty anchor tag like this. Since this is a placeholder container, it would be better for it to be a `<span>` that we will swap the content of.